commands/volumes/volume-create.go

```diff
@@ -120,6 +121,39 @@ func rollBackVolumeCreate(c *context.Context) error {
 	return nil
 }
 
+func registerVolCreateStepFuncs() {
+	var sfs = []struct {
+		name string
+		sf   transaction.StepFunc
+	}{
+		{"vol-create.Stage", validateVolumeCreate},
+		{"vol-create.Commit", generateVolfiles},
+		{"vol-create.Store", storeVolume},
+		{"vol-create.Rollback", rollBackVolumeCreate},
```

Comment:
I've just reused the existing functions here. But, you're correct. We will need to cleanup volfiles if any were generated.

We do need to remove the brick paths, as the volume create failed, and they are no longer useful. This allows them to be reused easily. We don't remove brick-paths when we're doing a volume-delete as it might still contain data.